src/Uno.UWP/Storage/Pickers/FileOpenPicker.cs

```diff
@@ -98,6 +99,27 @@ private void ValidateConfiguration()
 				throw new InvalidOperationException("You must provide at least a general file type filter ('*')");
 			}
 		}
+
+		internal void SetMultipleFileLimit(int limit)
+		{
+			_multipleFileLimit = limit;
+		}
+#endif
+	}
+
+	public static class FileOpenPickerExtensions
+	{
+
+		/// <summary>
+		/// Sets the file limit a user can select when picking multiple files.
+		/// </summary>
+		/// <param name="limit">The maximum number of files that the user can pick.</param>
+#if !__IOS__
+		[NotImplemented]
 #endif
+		public static void SetMultipleFilesLimit(this FileOpenPicker picker, int limit)
```

Comment:
Let's also document this in the Windows.Storage docs.

Reply:
docs were updated